<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1529</width>
    <height>927</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_girdiler">
       <attribute name="title">
        <string>Girdiler</string>
       </attribute>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>200</y>
          <width>134</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>R1FC Uzunluğu (mm) :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>131</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Kepçe Piston Çapı (mm) :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>64</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Piston Çalışma Basıncı (bar):</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>330</y>
          <width>134</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Kol Kütlesi (kg) :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>260</y>
          <width>134</width>
          <height>62</height>
         </rect>
        </property>
        <property name="text">
         <string>Kepçe Kütlesi (kg) :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>400</y>
          <width>134</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Boom Kütlesi (kg) :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="girdi_Dc">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>150</y>
          <width>408</width>
          <height>20</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhPreferNumbers</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="girdi_Pc">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>85</y>
          <width>408</width>
          <height>20</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhPreferNumbers</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="girdi_W_kepce">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>280</y>
          <width>408</width>
          <height>20</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhPreferNumbers</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="girdi_R1FC">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>220</y>
          <width>408</width>
          <height>20</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhPreferNumbers</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="girdi_W_kol">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>350</y>
          <width>408</width>
          <height>20</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhPreferNumbers</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="girdi_W_boom">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>420</y>
          <width>408</width>
          <height>20</height>
         </rect>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhPreferNumbers</set>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonHesapla">
        <property name="geometry">
         <rect>
          <x>860</x>
          <y>220</y>
          <width>162</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Hesapla</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>60</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>170 + - 70</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>130</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>65 + - 30</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>200</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>820 + - 200</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>260</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>729.46 + - 250</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>330</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>1500 + - 500</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>400</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>2100 + - 500</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_sonuc">
       <attribute name="title">
        <string>Hesap Sonuçları</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="tab_kepce_kepce">
           <attribute name="title">
            <string>Kepçe</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="maximumSize">
               <size>
                <width>511</width>
                <height>750</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resources.qrc">:/new/prefix1/png/kepce_serbest_diagram.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_16">
                  <property name="text">
                   <string>Maksimum kazma kuvveti (N) :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_maxFc">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_18">
                  <property name="text">
                   <string>Fperno1 x kuvveti (N) :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_F_P1x">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_17">
                  <property name="text">
                   <string>Fperno1 y kuvveti (N) :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_F_P1y">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_kepce_kol">
           <attribute name="title">
            <string>Kol</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="maximumSize">
               <size>
                <width>500</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resources.qrc">:/new/prefix1/png/kol_serbest_diagram.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>Kol Piston Kuvveti (Fb) (N) :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_Fb">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label_20">
                  <property name="text">
                   <string>Fperno 7 x (N) :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_F_P7x">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="label_22">
                  <property name="text">
                   <string>Fperno 7 y(N) :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_F_P7y">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QLabel" name="label_19">
                  <property name="text">
                   <string>Piston Çapı (mm) :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_Db">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_kepce_boom">
           <attribute name="title">
            <string>Boom</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_kepce_boom_serbest">
                <property name="maximumSize">
                 <size>
                  <width>400</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="resources.qrc">:/new/prefix1/png/boom_serbest_diagram.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_14">
                  <item>
                   <widget class="QLabel" name="label_24">
                    <property name="text">
                     <string>Boom Piston Kuvveti (Fa) (N) :</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_Fa">
                    <property name="text">
                     <string>0</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_13">
                  <item>
                   <widget class="QLabel" name="label_21">
                    <property name="text">
                     <string>Piston Çapı (mm) :</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_Da">
                    <property name="text">
                     <string>0</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_kepce_boom_serbest_eq">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>190</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>0</width>
                <height>10</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resources.qrc">:/new/prefix1/png/boom_serbest_diagram_eq.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1529</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
